cts/pengine/guest-node-host-dies.xml

```diff
@@ -1,10 +1,9 @@
-<cib crm_feature_set="3.0.10" validate-with="pacemaker-2.4" epoch="548" num_updates="7" admin_epoch="0" cib-last-written="Wed Mar 23 15:52:27 2016" update-origin="rhel7-2" update-client="crm_resource" update-user="root" have-quorum="1" dc-uuid="4">
+<cib crm_feature_set="3.0.10" validate-with="pacemaker-3.0" epoch="548" num_updates="7" admin_epoch="0" cib-last-written="Wed Mar 23 15:52:27 2016" update-origin="rhel7-2" update-client="crm_resource" update-user="root" have-quorum="1" dc-uuid="4">
   <configuration>
     <crm_config>
       <cluster_property_set id="cib-bootstrap-options">
         <nvpair id="cts-stonith-enabled" name="stonith-enabled" value="1"/>
         <nvpair id="cts-no-quorum-policy" name="no-quorum-policy" value="stop"/>
-        <nvpair id="cts-expected-quorum-votes" name="expected-quorum-votes" value="5"/>
```

Comment:
Doh! I obviously forgot to check for expected-quorum-votes, pcmk_arg_map, and notification-*. Generally I updated all the tests as I changed the feature support ...